lldb/include/lldb/API/SBProcess.h

```diff
@@ -160,6 +160,8 @@ class LLDB_API SBProcess {
 
   lldb::SBError Continue();
 
+  lldb::SBError ReverseContinue();
```

Comment:
For programming uses, it might be more useful to have one API with a bool or RunDirection or something? It's nicer to write:

`  process.Continue(direction)
`
rather than:

  if (direction = lldb.eRunBackwards):
    process.ReverseContinue()
  else:
    process.Continue()

Reply:
Yeah OK.